File: Python/PeralatanPetShop.py
class PeralatanPetShop:
    # constructor
    def __init__(self, id, nama, kategori, harga):
        self.id = id
        self.nama = nama
        self.kategori = kategori
        self.harga = harga

    # getter
    def get_id(self):
        return self.id

    def get_nama(self):
        return self.nama

    def get_kategori(self):
        return self.kategori

    def get_harga(self):
        return self.harga

    # setter
    def set_nama(self, nama):
        self.nama = nama

    def set_kategori(self, kategori):
        self.kategori = kategori

    def set_harga(self, harga):
        self.harga = harga

File: PHP/PeralatanPetShop.php
<?php
class PeralatanPetShop {
    private $id;
    private $nama;
    private $kategori;
    private $harga;
    private $foto; // path to image

    // constructor
    // untuk beberapa values yang tidak diisi (null), maka akan diisi dengan default value
    public function __construct($id, $nama, $kategori, $harga, $foto) {
        $this->id       = $id; // id harus diisi (required pada form), tidak akan null
        $this->nama     = $nama ? $nama : "tidak ada nama";
        $this->kategori = $kategori ? $kategori : "lainnya";
        $this->harga    = $harga ? $harga : 0;
        $this->foto     = $foto; // tidak ada default value, karena foto bisa kosong
    }

    // getters
    public function getId() {
        return $this->id;
    }

    public function getNama() {
        return $this->nama;
    }

    public function getKategori() {
        return $this->kategori;
    }

    public function getHarga() {
        return $this->harga;
    }

    public function getFoto() {
        return $this->foto;
    }

    // setters
    // note: tidak diperlukan set id karena id merupakan primary key
    public function setNama($nama) {
        $this->nama = $nama;
    }

    public function setKategori($kategori) {
        $this->kategori = $kategori;
    }

    public function setHarga($harga) {
        $this->harga = $harga;
    }

    public function setFoto($foto) {
        $this->foto = $foto;
    }
}
